test/e2e/reaper_test.go

```diff
@@ -82,30 +82,30 @@ func createMultiReaper(t *testing.T, ctx context.Context, namespace string, f *f
 	uiSecretKey := types.NamespacedName{Namespace: namespace, Name: "reaper-ui-secret"}
 	kcKey := types.NamespacedName{Namespace: namespace, Name: "test"}
 
-	checkSecretExists(t, f, ctx, kcKey, framework.ClusterKey{K8sContext: f.K8sContext(0), NamespacedName: cqlSecretKey})
-	checkSecretExists(t, f, ctx, kcKey, framework.ClusterKey{K8sContext: f.K8sContext(1), NamespacedName: cqlSecretKey})
-	checkSecretExists(t, f, ctx, kcKey, framework.ClusterKey{K8sContext: f.K8sContext(0), NamespacedName: jmxSecretKey})
-	checkSecretExists(t, f, ctx, kcKey, framework.ClusterKey{K8sContext: f.K8sContext(1), NamespacedName: jmxSecretKey})
-	checkSecretExists(t, f, ctx, kcKey, framework.ClusterKey{K8sContext: f.K8sContext(0), NamespacedName: uiSecretKey})
-	checkSecretExists(t, f, ctx, kcKey, framework.ClusterKey{K8sContext: f.K8sContext(1), NamespacedName: uiSecretKey})
+	checkSecretExists(t, f, ctx, kcKey, framework.ClusterKey{K8sContext: f.DataPlaneContexts[0], NamespacedName: cqlSecretKey})
```

Comment:
It is unnecessary to do all the secrets verification in the e2e tests. It can be done in the integrations more easily and more quickly. I realize that is unrelated to the changes in the PR, and I not suggesting such changes for this PR. Just wanted to share while it was on my mind :)

Reply:
Well let's remove it now, otherwise we will forget about it... again (I think you already mentioned that this verification was unnecessary).